rca/projects/models.py

```diff
@@ -1,9 +1,222 @@
-from rca.utils.models import BasePage
+from collections import defaultdict
 
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    ObjectList,
+    StreamFieldPanel,
+    TabbedInterface,
+)
+from wagtail.core.blocks import RichTextBlock
+from wagtail.core.fields import StreamField
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+from wagtail.images import get_image_model_string
+from wagtail.images.edit_handlers import ImageChooserPanel
 
-class ProjectPickerPage(BasePage):
-    pass
+from rca.home.models import HERO_COLOUR_CHOICES, LIGHT_TEXT_ON_DARK_IMAGE
+from rca.programmes.models import Subject
+from rca.utils.blocks import (
+    AccordionBlockWithTitle,
+    GalleryBlock,
+    LinkBlock,
+    QuoteBlock,
+)
+from rca.utils.models import BasePage, RelatedStaffPageWithManualOptions
+
+
+class ProjectPageSubjectPlacement(models.Model):
+    page = ParentalKey("ProjectPage", related_name="subjects")
+    subject = models.ForeignKey(
+        Subject,
+        on_delete=models.SET_NULL,
```

Comment:
It might be better to make this `CASCADE` so orphan placement records aren't left. Also remove `blank=False` and `null=True`